src/pages/portal/UserPortal.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Container } from '@mui/system';
+import List from '@mui/material/List';
+import Typography from '@mui/material/Typography';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+
+
+
+export default function UserPortal() {
+
+    const listElements = [
+        {
+            id: "",
+            name: "Last name: ",
+            address: "Address: ",
+            phone: "Phone: ",
+            email: "Email: ",
+            loanAmount: "Loan amount: ",
+            instalment: "Number of months to pay: ",
+            total: "Total amount to be paid: "
```

Comment:
The field names here aren't all correct in terms of what the API sends back to us. Also be careful of typos. We need to look at the back-end code to see the fieldNames it sends to the front end and use those.